packages/@dcl/sdk-commands/package.json

```diff
@@ -17,9 +17,11 @@
   "license": "Apache-2.0",
   "dependencies": {
     "@dcl/dcl-rollup": "file:../dcl-rollup",
+    "@dcl/ecs": "file:../ecs",
     "@dcl/hashing": "1.1.3",
     "@dcl/mini-comms": "1.0.1-20230216163137.commit-a4c75be",
-    "@dcl/protocol": "1.0.0-4114477251.commit-ccb88d6",
+    "@dcl/protocol": "https://sdk-team-cdn.decentraland.org/@dcl/protocol/branch//dcl-protocol-1.0.0-4319716033.commit-a936923.tgz",
```

Comment:
I think we should keep the datalayer rpc .proto files inside this repo! it will greatly improve the developer experience